### PR TITLE
fix(docs): let CodeSandbox package versions match our versions

### LINKED ISSUE #1607: Opening Typescript CodeSandbox for Chat broken
## Bug Report
Opening the first chat example as a Typescript CodesandBox immediately fails:
image

### Steps
1. Navigate to https://stardust-ui.github.io/react/components/chat#types-chat
2. Hit "Try It" on first chat example
3. Switch to Typescript
4. Generate and open CodeSandbox

### Expected Result
CodeSandbox runs correctly

### Actual Result
CodeSandbox crashes

### Version
0.34.0

### PR DESCRIPTION
**Pull request by lucivpav**
Fixes #1607 

### Before
image

### Now
image

### CODE REVIEW

File: docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts [hunk 1]
```diff
@@ -3,20 +3,23 @@ import * as _ from 'lodash'
...
+const packageJson = require('../../../../../../package.json')
+const codeSandboxPackageJson = require('../../../../../../packages/code-sandbox/package.json')
+
 const name = 'stardust-ui-example'
 const description =
   'An exported example from Stardust UI React, https://stardust-ui.github.io/react/'
 const dependencies = {
-  '@stardust-ui/code-sandbox': 'latest',
-  ..._.mapValues(imports, () => 'latest'),
+  '@stardust-ui/code-sandbox': codeSandboxPackageJson.version,
```
Comment (layershifter) [inline docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts]: We can use `latest` for this, it was designed for it.

File: docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts [hunk 2]
```diff
@@ -3,20 +3,23 @@ import * as _ from 'lodash'
...
+  ..._.mapValues(imports, (value, key, object) => value.version),
```
Comment (layershifter) [inline docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts]:
```suggestion
  ..._.mapValues(imports, (package) => package.version),
```

File: docs/src/components/Playground/renderConfig.ts [hunk 3]
```diff
@@ -17,13 +21,34 @@ export const babelConfig = {
...
+  '@stardust-ui/react': {
+    version: stardustReactPackageJson.version,
+    module: Stardust,
+  },
+  classnames: {
+    version: stardustReactPackageJson.dependencies['classnames'],
+    module: Classnames,
+  },
+  lodash: {
+    version: packageJson.devDependencies['@types/lodash'],
```
Comment (layershifter) [inline docs/src/components/Playground/renderConfig.ts]: Oops, should be

```suggestion
    version: stardustReactPackageJson.dependencies['lodash'],
```

File: docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts [hunk 4]
```diff
@@ -3,20 +3,23 @@ import * as _ from 'lodash'
...
   // https://github.com/stardust-ui/react/issues/1519
   'react-scripts': 'latest',
 }
 const devDependencies = {
-  '@types/lodash': 'latest',
-  '@types/react': 'latest',
-  '@types/react-dom': 'latest',
+  '@types/lodash': packageJson.devDependencies['@types/lodash'],
+  '@types/react': packageJson.devDependencies['@types/react'],
+  '@types/react-dom': packageJson.devDependencies['@types/react-dom'],
```
Comment (layershifter) [inline docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts]: Move it to `imports`?
https://github.com/stardust-ui/react/pull/1849/files#diff-4f87c22dac940c3273a45733a5faf28d

Comment (lucivpav) [reply docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts]: All of these are already in `imports`. Removing them now.

File: docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts (hunk 2 above)
Comment (lucivpav) [reply docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts]: `package` is a reserved word in strict mode FYI

File: docs/src/components/Playground/renderConfig.ts [hunk 5]
```diff
@@ -17,13 +21,34 @@ export const babelConfig = {
...
+  '@stardust-ui/react': {
+    version: stardustReactPackageJson.version,
+    module: Stardust,
+  },
+  classnames: {
+    version: stardustReactPackageJson.dependencies['classnames'],
+    module: Classnames,
+  },
+  lodash: {
+    version: stardustReactPackageJson.devDependencies['@types/lodash'],
```
Comment (layershifter) [inline docs/src/components/Playground/renderConfig.ts]:
```suggestion
    version: stardustReactPackageJson.dependencies['lodash'],
```

Oops, still points to `@types/`

File: docs/src/components/Playground/renderConfig.ts [hunk 6]
```diff
@@ -17,13 +21,34 @@ export const babelConfig = {
...
+    module: _,
+  },
+  react: {
+    version: packageJson.devDependencies['react'],
+    module: React,
+  },
+  'react-dom': {
+    version: packageJson.devDependencies['react-dom'],
```
Comment (layershifter) [inline docs/src/components/Playground/renderConfig.ts]: We can get this from `peerDependencies` in `stardustReactPackageJson`:

```suggestion
    version: packageJson.peerDependencies['react-dom'],
```

File: docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts [hunk 7]
```diff
@@ -8,16 +8,12 @@ const description =
...
   // required to enable all features due old templates in https://github.com/codesandbox/codesandbox-importers
   // https://github.com/stardust-ui/react/issues/1519
   'react-scripts': 'latest',
 }
-const devDependencies = {
-  '@types/lodash': 'latest',
-  '@types/react': 'latest',
-  '@types/react-dom': 'latest',
-}
+const devDependencies = {}
```
Comment (layershifter) [inline docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts]: Can be removed at all, now

File: docs/src/components/Playground/renderConfig.ts [hunk 8]
```diff
@@ -17,13 +20,34 @@ export const babelConfig = {
 }
 
 export const imports = {
```
Comment (layershifter) [inline docs/src/components/Playground/renderConfig.ts]:
```suggestion
export const imports: Record<string, { version: string, module: any }> = {
```

One more thing, can we introduce a type for this?

**Review by layershifter — APPROVED** [review]

**Pull request merged**